cmd/vic-machine/common/dns.go

```diff
@@ -0,0 +1,56 @@
+package common
```

Comment:
I have changed it so that it puts in 2017. However, we should not modify the make target since its runs on CI, and we don't want to modify files during builds.